src/OSPSuite.Presentation/Presenters/ContextMenus/ClassificationNodeContextMenu.cs

```diff
@@ -20,15 +20,23 @@ protected ClassificationNodeContextMenu(ClassificationNode objectRequestingConte
 
       protected override IEnumerable<IMenuBarItem> AllMenuItemsFor(ClassificationNode classificationNode, TPresenter presenter)
       {
-         yield return RenameGroupMenuFor(classificationNode, presenter);
```

Comment:
I am not much of a yield fan anymore. It make some simple stuff too hard. Back to basics for simple like this